src/components/data_display/generateFilterDataFields/generateFilterDataFields.ts

```diff
@@ -138,7 +138,7 @@ export const generateFilterDataFields = async (table: string, fullUpdate: boolea
               // TO-DO: Need to pass Auth headers to evaluator API calls
               graphQLConnection: { fetch, endpoint: graphQLEndpoint },
             })
-            patch[camelCase(column)] = evaluatedResult
+            patch[camelCase(column)] = evaluatedResult === '' ? null : evaluatedResult
```

Comment:
This is a bit hacky, but it's much better in most cases if empty strings get stored as `null`. There may be a situation where we *don't* want this, but I can't think of one, so this is probably simplest for now.